### DISCUSSION TITLE
Describe how to use labels (apply to feed or article?)

### DISCUSSION THREAD

**Discussion by coderofsalvation** [FreshRSS/FreshRSS#5921] [Q&A]
Hi first of all thanks for this amazing software.
It almost seems its a RSS reader and webscraper in one.
from the documentation it is not clear to me how I can assign a label to a feed or article.
The docs talk about how to use them in filters yes, but that's it.
I know how to 'manage labels', but I don't know how to use them :/

any tips in the meanwhile?

**Comment by Alkarex** [comment]
See the menu in the footer of an article. You can create new labels there or click to assign existing ones to the current article

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/a6abb6cf-f96c-4966-9e1d-ba6078922381)
